libs/api/domains/watson-assistant-chat/tsconfig.spec.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../../../dist/out-tsc",
+    "module": "commonjs",
+    "types": ["jest", "node"]
+  },
+  "include": [
```

Comment:
FYI the include array is generated as `"include": ["**/*.test.ts", "**/*.spec.ts", "**/*.d.ts"]` from the nest library generator. As `.tsx` should be redundant in gql api library.

Reply:
I tweaked the tsconfig file since I generated the project as a node project